apps/deepsirius-ui/src/server/api/routers/ssh.ts

```diff
@@ -110,16 +117,24 @@ export const sshRouter = createTRPCRouter({
           Cookie: cookie,
         },
       });
+      if (!res.ok) {
+        const data: unknown = await res.json();
+        const error = apiErrorSchema.parse(data);
 
+        throw new TRPCError({
+          code: 'INTERNAL_SERVER_ERROR',
+          message: error.message,
+        });
+      }
       const data = await res.text();
       if (!res.ok) {
```

Comment:
Redundant if statement?

Reply:
Thanks for catching these unnecessary catches. Should we add this to a patch?